tests/fuzzer/fuzzer.c

```diff
@@ -0,0 +1,70 @@
+/**
```

Comment:
Should this test live inside framework, like the selftests and MCE check?

Reply:
You're right, it is better suited inside framework.